azure_functions_worker/dispatcher.py

```diff
@@ -62,17 +63,11 @@ def __init__(self, loop: BaseEventLoop, host: str, port: int,
 
         self._old_task_factory = None
 
-        # A thread-pool for synchronous function calls.  We limit
-        # the number of threads to 1 so that one Python worker can
-        # only run one synchronous function in parallel.  This is
-        # because synchronous code in Python is rarely designed with
-        # concurrency in mind, so we don't want to allow users to
-        # have races in their synchronous functions.  Moreover,
-        # because of the GIL in CPython, it rarely makes sense to
-        # use threads (unless the code is IO bound, but we have
-        # async support for that.)
+        # We allow the customer to change synchronous thread pool count by
+        # PYTHON_THREADPOOL_THREAD_COUNT app setting. The default value is 1.
+        thread_count = get_app_setting(PYTHON_THREADPOOL_THREAD_COUNT, '1')
```

Comment:
Currently, this is only sync threads - make this explicit in the variable name.

```suggestion
        sync_tp_max_workers_count = get_app_setting(PYTHON_THREADPOOL_THREAD_COUNT, '1')
```

Reply:
Also, make it `int` here itself